src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -92,14 +92,20 @@ const DomainFragment = ({
 
     navigation.push('DomainScreen', navigationParam);
   };
+  const handleScroll = React.useCallback(() => {
+    Keyboard.dismiss();
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleScroll` function is a good addition. It will dismiss the keyboard when the user starts scrolling, which can improve the user experience by hiding the keyboard when it's not needed.
```diff
+  const handleScroll = React.useCallback(() => {
+    Keyboard.dismiss();
+  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->